File: C++/is_sorted2.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int n;
    cin >> n;
    vector<int> nums(n);
    for (int i=0; i<n; i++){
        cin >> nums[i];
    }

    int drop_ctr = 0;

    for(int i=0; i<n; i++){
        if (i==0){
            if (nums[i] < nums[n-1]){
                drop_ctr++;
            }
        }
        else if (nums[i] < nums[i-1]){
            drop_ctr++;
        }
    }
    if (drop_ctr <= 1){
        cout << "True" << endl;
    }
    else{
        cout << "False" << endl;
    }
    return 0;
}

File: Python/is_sorted2.py
n = int(input())
nums = []
for _ in input().split():
    nums.append(int(_))
drop_ctr = 0
for i in range(0,n):
    if nums[i] < nums[i-1]:
        drop_ctr += 1

if drop_ctr <= 1:
    print("True")
else:
    print("False")
